docs/editors/helix.md

```diff
@@ -0,0 +1,26 @@
+---
+id: helix
+sidebar_label: Helix
+title: Helix
+---
+
+![helix demo](https://i.imgur.com/b0sETIY.gif)
+
+## Installation
+
+Helix requires `metals` to be on the user's path, it should then be
+automatically detected and enabled. Installing metals in this manner is most
+readily achieved via [Coursier](https://get-coursier.io/), running
+`coursier install metals` should be sufficient. To check that Helix is able to
+detect metals after this run `hx --health scala`
+
+## Importing builds
```

Comment:
Could be a good idea to describe here the new auto import Metals feature and how to set it up in Helix. :wink: Might be also helpful to mention the new LSP Progress integration (to workaround the https://github.com/helix-editor/helix/issues/5524), although that one might be already enabled by default if not otherwise set, right? :slightly_smiling_face:

Reply:
Yea the progress stuff is enabled by default so no need to explicitly document I don't think.  The auto import stuff isn't merged yet but can come back and add it when it is.